pkg/scripts/os.go

```diff
@@ -246,22 +242,65 @@ RELEASE="v{{ .KUBERNETES_VERSION }}"
 sudo mkdir -p /var/tmp/kube-binaries
 cd /var/tmp/kube-binaries
 sudo curl -L --remote-name-all \
-	https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
+	https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/kubeadm
 
 sudo mkdir -p /opt/bin
 cd /opt/bin
 sudo systemctl stop kubelet
-sudo mv /var/tmp/kube-binaries/{kubeadm,kubelet,kubectl} .
-sudo chmod +x {kubeadm,kubelet,kubectl}
-
-curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/kubelet.service" |
-	sed "s:/usr/bin:/opt/bin:g" |
-	sudo tee /etc/systemd/system/kubelet.service
+sudo mv /var/tmp/kube-binaries/kubeadm .
+sudo chmod +x kubeadm
 
 sudo mkdir -p /etc/systemd/system/kubelet.service.d
 curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/10-kubeadm.conf" |
 	sed "s:/usr/bin:/opt/bin:g" |
 	sudo tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+
+sudo systemctl daemon-reload
+sudo systemctl start kubelet
```

Comment:
Even that this is related to `kubeadm`, I guess this should go to another script (kubelet-related one).

Reply:
definitely

Reply:
Fixed and rebased the PR.